### DISCUSSION TITLE
Based on the image given why is the 1900 number unnecessarily coming on my github page and how to remove it?

### DISCUSSION THREAD

**Discussion by Nuza-Nigar** [community/community#77555] [Programming Help]
### Select Topic Area

Question

### Body

How to remove 1900 on my github page. Which file should I check based on inspection data? 

![Screenshot_16](https://github.com/community/community/assets/150617190/22d432c5-baba-4f86-94f2-3b5b0d4544b2)

**Comment by leeringrose** [comment]
I had a look around on your GitHub page and didn't see any 1900 on your GitHub page. The screenshot you shared here is not for Github page, so I recommend you to check your header component or element if it's unexpectedly added to any gaps of it's code scripts.
If you are not an owner of code, could you please drop the link to this webpage where I can refer to?

**Comment by Bandaranayaka** [comment]
If you can provide more specific details or context, I might be able to offer more targeted advice.
